parsers/fortran.c

```diff
@@ -582,10 +582,11 @@ static void makeLabelTag (vString *const label)
 
 static lineType getLineType (void)
 {
-	vString *label = vStringNew ();
+	static vString *label = NULL;
```

Comment:
It's a "leak" in the sense that memory is allocated by the application and never released by it. "Reachable" is just a type of leak that still has a pointer pointing to the leaked memory at the end of the process.